micro_sam/util.py

```diff
@@ -145,6 +142,36 @@ def _get_device(device):
     return device
 
 
+def _get_device(device=None):
```

Comment:
I wanted better error handling, if users selected a device that wasn't available on their system.
Then I thought I'd go one step better, and only display the available devices int he embedding widget dropdown menu (using the new `_get_available_devices()` function).

Reply:
Looks good!